llvm/test/CodeGen/PowerPC/vsx-fma-m-early.ll

```diff
@@ -0,0 +1,77 @@
+; RUN: llc -verify-machineinstrs -mcpu=pwr10 -disable-ppc-vsx-fma-mutation=false -ppc-asm-full-reg-names -schedule-ppc-vsx-fma-mutation-early < %s | \
```

Comment:
- Do we need to consider more than just P10 (P8, P9 for example) and Linux? If Linux needs to be considered, we probably should specify the triple in the RUN line rather than in the IR itself.
- The RUN line looks a bit too long, we should limit this to 80 characters.

Reply:
it just for power10, The `xxspltiw`  is introduced in power10 , I think linux has the same problem, but linux will generate a different asm instructions set . I think testing in AIX is enough for the code change